### PR TITLE
Rework

### PR DESCRIPTION
**Pull request by Maxopoly**

### CODE REVIEW

File: src/com/untamedears/ItemExchange/utility/ExchangeRule.java [hunk 1]
```diff
@@ -305,12 +306,12 @@ private static String showString(String string) {
...
 
 	/*
-	 * Adds a � in front of every character in a string
+	 * Adds a � in front of every character in a string
 	 */
 	private static String hideString(String string) {
 		String hiddenString = "";
 		for (char character : string.toCharArray()) {
-			hiddenString += "�" + character;
+			hiddenString += ChatColor.translateAlternateColorCodes('-',"-") + character;
```
Comment (ProgrammerDan) [inline src/com/untamedears/ItemExchange/utility/ExchangeRule.java]: Just use `ChatColor.COLOR_CHAR` here

Comment (ProgrammerDan) [inline src/com/untamedears/ItemExchange/utility/ExchangeRule.java]: Just use `ChatColor.COLOR_CHAR` here

**Comment by ProgrammerDan** [comment]
So I'm inclined to agree this might break everything, but who knows!

I understand what they are using now to "compile" the rule; it's a very clever hack leveraging the auto-hiding of special characters.

**Comment by Maxopoly** [comment]
Let's try and if it breaks we revert it and return back to the default behavior, where we just pretend this plugin doesn't exist.

**Comment by ProgrammerDan** [comment]
Totally down with it

**Pull request merged**